src/libraries/Microsoft.PowerFx.Connectors/Execution/HttpFunctionInvoker.cs

```diff
@@ -63,50 +60,97 @@ public HttpRequestMessage BuildRequest(FormulaValue[] args)
+                }
+                else if (map.TryGetValue(param.Name, out var paramValue))
                 {
                     var valueStr = paramValue.ToObject().ToString();
 
-                    switch (param.In.Value)
+                    if (param.In == null)
                     {
-                        case ParameterLocation.Path:
-                            path = path.Replace("{" + param.Name + "}", HttpUtility.UrlEncode(valueStr));
-                            break;
-
-                        case ParameterLocation.Query:
-                            query.Append((query.Length == 0) ? "?" : "&");
-                            query.Append(param.Name);
-                            query.Append('=');
-                            query.Append(HttpUtility.UrlEncode(valueStr));
-                            break;
-
-                        case ParameterLocation.Header:
-                            headers.Add(param.Name, valueStr);
-                            break;
-
-                        default:
-                            throw new NotImplementedException($"{param.In}");
+                        // No schema, we send the raw value
+                        body = paramValue.ToObject().ToString();
                     }
+                    else
+                    {
+                        switch (param.In.Value)
+                        {
+                            case ParameterLocation.Path:
+                                path = path.Replace("{" + param.Name + "}", HttpUtility.UrlEncode(valueStr));
+                                break;
+
+                            case ParameterLocation.Query:
+                                query.Append((query.Length == 0) ? "?" : "&");
+                                query.Append(param.Name);
+                                query.Append('=');
+                                query.Append(HttpUtility.UrlEncode(valueStr));
+                                break;
+
+                            case ParameterLocation.Header:
+                                headers.Add(param.Name, valueStr);
+                                break;                           
+
+                            case ParameterLocation.Cookie:
+                            default:
+                                throw new NotImplementedException($"{param.In}");
+                        }
+                    }                    
                 }
             }
 
             var url = path + query.ToString();
 
             // $$$ Not handling Body yet...
-            var request = new HttpRequestMessage(_method, url);
+            var request = new HttpRequestMessage(_method, url);            
 
             foreach (var kv in headers)
             {
                 request.Headers.Add(kv.Key, kv.Value);
             }
 
+            if (!string.IsNullOrEmpty(body))
+            {                
+                // $$$ Might need System.Net.Http.FormUrlEncodedContent()
```

Comment:
>// $$$ Might need System.Net.Http.FormUrlEncodedContent() [](http://example.com/codeflow?start=16&length=57)

Temporary comment, I think we can remove it
As we serialize the content to a string, calling StringContent is enough and we provide the right content type